LocalFolder/explainer.md

```diff
@@ -0,0 +1,206 @@
+        value.remove({ recursive: true });
+    }
+```
+
+## Similar storage folders
+
+There are other WinRT storage APIs similar to LocalFolder that can be exposed the same way through OPFS directory handles, but are not currently planned to be supported.
+
+* [LocalCacheFolder](https://learn.microsoft.com/en-us/uwp/api/windows.storage.applicationdata.localfolder?view=winrt-22621)
+* [RoamingFolder](https://learn.microsoft.com/en-us/uwp/api/windows.storage.applicationdata.roamingfolder?view=winrt-22621)
+* [SharedLocalFolder](https://learn.microsoft.com/en-us/uwp/api/windows.storage.applicationdata.sharedlocalfolder?view=winrt-22621)
+* [TemporaryFolder](https://learn.microsoft.com/en-us/uwp/api/windows.storage.applicationdata.temporaryfolder?view=winrt-22621)
+
+## Alternatives considered
+
+### New Web API
+
+We considered creating a new API on `navigator` to return a `FileSystemDirectoryHandle` of the app's LocalFolder directory. This was rejected primarily because of the long term cost of maintaining a non-standard API. 
+
+### File migration
+
+We considered a solution that migrates the contents of the LocalFolder directory into OPFS storage. The drawbacks to this solution includes:
+
+* It could take a long time to migrate large files. There is no way for the calling code to register a callback to wait for migration to be completed without introducing a new API, negating the advantage of not creating a new API on `navigator`.
+* Migration of files could fail due the lack of storage space and other I/O errors. There is no good way to communicate such errors to the calling code without introducing a new API.
+
+## Security risks 
+
+1. A malicious party could attempt to gain access to files outside of the boundary of the `LocalFolder` directory.
+
+1. A malicious party could create executable files on the file system.
+
+1. Third party scripts could read data in `LocalFolder` even though the app developer is not aware of this feature. 
+
+1. Scripts in embedded iframes could try to access data in `LocalFolder`. 
+
+### Mitigations
+
+1. The `FileSystemHandle` design does not support walking up the directory structure.
+
+1. The `LocalFolder` directory handle and its contents will not allow the modification of existing files or creation of new files. This prevents the creation of executable files and the creation of files outside of the LocalFolder directory. Additionally, the implementation of the File System Access API in Chromium prevents creation of some executable file types.
+
+1. The app has to opt in explictly using the `related_applications` field in the web app manifest. The app developer must understand the risks involved and be responsible for the actions of third party scripts used. Most apps that do not intend to access `LocalFolder` will not expose it unknowingly to third parties.
+
+1. The `LocalFolder` handle entry will not be visible to iframes as OPFS storage is partitioned by origin.
```

Comment:
Does this need an additional qualifier about what type of iframe it won't be visible to? e.g. "will not be not visible to cross-origin iframes"?